app/src/main/res/layout/item_selection_interaction_items.xml

```diff
@@ -27,13 +27,13 @@
       android:id="@+id/item_selection_checkbox"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      android:checked="@{viewModel.answerSelected}"
+      app:buttonTint="@{viewModel.isEnabled ? @color/component_color_shared_item_selection_interaction_enabled_color : @color/component_color_shared_item_selection_interaction_disabled_color}"
       android:clickable="false"
       android:enabled="@{viewModel.isEnabled}"
       android:focusable="false"
       android:labelFor="@id/item_selection_contents_text_view"
-      app:buttonTint="@{viewModel.isEnabled ? @color/component_color_shared_item_selection_interaction_enabled_color : @color/component_color_shared_item_selection_interaction_disabled_color}" />
-
+      android:checked="@{viewModel.answerSelected}"
+      />
```

Comment:
Is the formatting correct?

Reply:
I intentionally did that because checkboxes weren't showing the checked state when the user rotated the device. After investigating for days, I found that it was because of the `buttontint `binding adapter. After changing the position of `checked `and `buttontint`, it works."

Reply:
That sounds good.

Re: formatting - isn't the closing tag `/>` supposed to be at the end of the previous line 35?

Reply:
Done.